fluent-bundle/src/memoizer.rs

```diff
@@ -7,10 +7,11 @@ pub trait MemoizerKind: 'static {
     where
         Self: Sized;
 
-    fn with_try_get<I, R, U>(&self, args: I::Args, cb: U) -> Result<R, ()>
+    fn with_try_get_threadsafe<I, R, U>(&self, args: I::Args, cb: U) -> Result<R, I::Error>
```

Comment:
this function is necessary to make the pluralrules thing work, and it's fine since we know that `I=PluralRules` is Send/Sync. An alternate way of doing this is to write this as `with_pluralrules`